radicle-httpd/src/api/v1/projects.rs

```diff
@@ -393,10 +393,10 @@ async fn issues_handler(
     let storage = &ctx.profile.storage;
     let repo = storage.repository(project)?;
     let issues = Issues::open(ctx.profile.public_key, &repo)?;
+    let mut issues: Vec<_> = issues.all()?.filter_map(|r| r.ok()).collect::<Vec<_>>();
```

Comment:
You should probably do the `skip` and `take` before sorting, so you minimise the elements to sort.

Reply:
That would be nice, but the thing is that if we skip and take before sorting, we'll have elements that will be excluded from the final output.
I'm aware of the performance hit, but else we won't be able to present the issues or patches in a sorted way to the user.

Reply:
Ah yes, you're dead right! :)